Classes/WPEditorViewController.m

```diff
@@ -1555,13 +1589,25 @@ - (void)editorView:(WPEditorView*)editorView
 - (void)editorView:(WPEditorView*)editorView
       fieldFocused:(WPEditorField*)field
 {
-    if (!field || field == self.editorView.titleField) {
+    if (field == self.editorView.titleField) {
```

Comment:
Was testing this to see if it was possible to see the toolbar items when no field was focused, but of course it isn't.  I can see how this is necessary for source mode to be able to show the toolbar items.

Nice work. :+1: